apps/dashboard/src/components/primitives/field-editor/variable-popover/variable-popover.tsx

```diff
@@ -0,0 +1,257 @@
+              <FormControl>
+                <div className="grid gap-1">
+                  <label className="text-text-sub text-label-xs">Variable name</label>
+                  <InputField size="fit" className="min-h-0">
+                    <Code2 className="text-text-sub h-4 w-4" />
+                    <Input value={name} onChange={(e) => handleNameChange(e.target.value)} className="h-7 text-sm" />
+                  </InputField>
+                </div>
+              </FormControl>
+            </FormItem>
+            <FormItem>
+              <FormControl>
+                <div className="grid gap-1">
+                  <label className="text-text-sub text-label-xs">Default value</label>
+                  <InputField size="fit" className="min-h-0">
+                    <Input
+                      value={defaultVal}
+                      onChange={(e) => handleDefaultValueChange(e.target.value)}
+                      className="h-7 text-sm"
+                    />
+                  </InputField>
+                </div>
+              </FormControl>
+            </FormItem>
+            <FormItem>
+              <FormControl>
+                <div className="flex items-center justify-between">
+                  <label className="text-text-sub text-label-xs">Show raw liquid</label>
+                  <Switch checked={showRawLiquid} onCheckedChange={setShowRawLiquid} className="scale-75" />
+                </div>
+              </FormControl>
+            </FormItem>
+            {showRawLiquid && (
+              <FormItem>
+                <FormControl>
+                  <div className="grid gap-1">
+                    <InputField size="fit" className="min-h-0">
+                      <Input
+                        value={formatLiquidVariable(name, defaultVal, transformers)}
+                        onChange={(e) => handleRawLiquidChange(e.target.value)}
+                        className="h-7 text-sm"
+                      />
+                    </InputField>
+                  </div>
+                </FormControl>
+              </FormItem>
+            )}
+          </div>
+
+          <Separator className="my-0" />
+
+          <div className="flex flex-col gap-1 p-2">
+            <FormItem>
+              <FormControl>
+                <div className="grid gap-1">
+                  <label className="text-text-sub text-label-xs">Modifiers</label>
+                  <Select
+                    value=""
+                    onValueChange={(value) => {
+                      if (value) {
+                        handleTransformerToggle(value);
+                        setSearchQuery('');
+                      }
+                    }}
+                  >
+                    <SelectTrigger className="!text-paragraph-xs text-text-soft !h-7 min-h-0 !p-1.5">
+                      <SelectValue placeholder="Add a modifier..." />
+                    </SelectTrigger>
+                    <SelectContent
```

Comment:
the content should be the side of the select input and look like a dropdown and not another popover

![Screenshot 2024-12-31 at 13 19 53](https://github.com/user-attachments/assets/41f28dc5-d5be-4d40-a2d7-55a9e422ef0e)

Reply:
on search it changes it's size

https://github.com/user-attachments/assets/c19534ce-5b20-4ae1-b811-92805b63ee02

Reply:
when clicking outside of the modifiers it closes the variable popover 

https://github.com/user-attachments/assets/0b212216-8dd7-4ead-a8a0-9e9e47102a5b